packages/utils/src/TestUtils.ts

```diff
@@ -52,6 +52,24 @@ class TestUtils {
     fn: (...args: TArgs) => TResult
   ): jest.Mock<TResult, TArgs> => (fn as unknown) as jest.Mock<TResult, TArgs>;
 
+  /**
+   * Selectively disable logging methods on `console` object. Uses spyOn so that
+   * changes will be reverted after leaving the test scope that it is set in. If
+   * no method names are given, all will be disabled.
+   */
```

Comment:
I would maybe change this to a single param that takes an array and then clarify the intention w/ `@param methodNames The console methods to disable`

Reply:
> 

There are a few prop type warning / errors in existing tests which might be good candidates. The specific use case though is in `setupTests.js` I added a `console.error` to surface hook errors that are otherwise swallowed. For cases where we expect an error, this is a way to disable.

Reply:
Good catch. I will add the `@param methodNames`. Is there a strong reason not to use the rest params though? Looks like it works fine either way.

<img width="754" alt="image" src="https://github.com/deephaven/web-client-ui/assets/1900643/73d5f5d9-1b7e-4439-9154-a9881a8001fa">

Reply:
The main benefit would be adding params in the future is easier if needed, but I don't expect this to be used too often.

I think part of my tendency also stems from before `...rest` existed and you had to use `arguments` as a magic value if you wanted to use variadic functions

Reply:
I'll probably just add the jsdoc on this one and leave the rest param unless you feel strongly otherwise